Multisig/Logic/Ledger/LedgerController.swift

```diff
@@ -8,17 +8,54 @@
 
 import Foundation
 
-class LedgerCommandController {
-    static func getAddressCommand(path: String,
-                                  displayVerificationDialog: Bool = true,
-                                  chainCode: Bool = false) -> Data {
+class LedgerController {
+    let bluetoothController: BluetoothController
+
+    init(bluetoothController: BluetoothController) {
+        self.bluetoothController = bluetoothController
+    }
+
+    struct LedgerInfo {
+        let address: Address
+        let path: String
+        let name: String
+    }
+
+    typealias LedgerKeyInfoCompletion = (LedgerInfo?) -> Void
+
+    func getAddress(deviceId: UUID, at index: Int, completion: @escaping LedgerKeyInfoCompletion) {
+        guard let device = bluetoothController.deviceFor(deviceId: deviceId) else {
+            completion(nil)
+            return
+        }
+        let path = "44'/60'/0'/0/\(index)"
```

Comment:
you should supply path as an argument

Reply:
it will be mitigated in following PRs :)